status-bar/android/src/main/java/com/capacitorjs/plugins/statusbar/StatusBarConfig.java

```diff
@@ -0,0 +1,34 @@
+package com.capacitorjs.plugins.statusbar;
+
+import com.getcapacitor.util.WebColor;
+
+public class StatusBarConfig {
```

Comment:
Fully up to you, but this file could have been Kotlin if you wanted. We should check, but I think fully new files can be Kotlin. Someone correct me if I'm wrong here.

Reply:
We need to add Kotlin support in the build.gradle first I think.

Reply:
No, Kotlin is already supported.  However, I don't think it makes sense to do it now.  We can make the Java -> Kotlin conversion its own PR.